### ISSUE TITLE
Include ormar ORM in FastAPI documentation

### ISSUE THREAD

**Issue by collerek** [tiangolo/fastapi#3392]
Hi!

Some users or `ormar` approached me and asked me why `ormar` is not included in `FastAPI` documentation, as they wished they knew about it sooner.

And I answered that I am not sure if including more orms is welcomed as I kind of cannot find a key what and why is included. 
I mean I get it why `sqlalchemy` and `encode/databases` (most popular sync orm (well since 1.4 also async) and most popular async repo), but why there is `Peewee` but not `Gino` or `Tortoise`?

[`ormar`](https://github.com/collerek/ormar) originates from [`encode/orm`](https://github.com/encode/orm) and [`ormantic`](https://github.com/awesometoolbox/ormantic) (both unmaintained) and build on top of them. The goal was to create a simple ORM that can be used directly (as request and response models) with FastAPI because ormar bases it's data validation on `pydantic` (each ormar model is also a pydantic one).

I can issue a PR for this but before I start I would like your input @tiangolo. 
Do you want to include more orms? 
Do you want to include `ormar`?

Let me know what you think! 
And thanks for an awesome job with FastAPI! :)

**Comment by ArcLightSlavik** [comment]
>but why there is Peewee but not Gino or Tortoise

Peewee was probably added because tiangolo (whoever created that doc was somehow lost to history 🤔 ) was working/interested in it (fun fact you should probably never use peewee with FastAPI, but that is a separeate PR). Gino/Tortoise on the other hand already have tutorials on connecting with FastAPI (https://python-gino.org/docs/en/master/tutorials/fastapi.html / https://tortoise-orm.readthedocs.io/en/latest/examples/fastapi.html), so I think no one was interested in making a PR that is just a duplicate of it.

I think for me the best would be having a tutorial for sqlalchemy and then a list of other orm's linking to there respective tutorials.

**Comment by collerek** [comment]
> Gino/Tortoise on the other hand already have tutorials

That's true, ormar have one too, but that way you need to find an orm and find out if it supports FastAPI, and in my opinion it should be other way around (check orms that works with FastAPI).

You already have a tutorial for sync sqlalchemy, and as async support is still in alpha phase I wouldn't expect it to pop up soon.

**Comment by collerek** [comment]
Ok, my bad 🤣
